src/main/java/uk/gov/hmcts/sptribs/ciccase/model/UserRole.java

```diff
@@ -31,7 +31,9 @@ public enum UserRole implements HasRole {
 
     ST_CIC_JUDGE("caseworker-st_cic-judge", "CRU"),
     ST_CIC_RESPONDENT("caseworker-st_cic-respondent", "CRU"),
-
+    CASEWORKER_ADMIN_PROFILE("hmcts-admin", "CRU"),
```

Comment:
Is **hmcts-admin** role access required for case flags?

Reply:
We added it as part of Case flags, maybe will try removing this role.

Reply:
just remembering, It might be needed for supplementary data. So, we might need this, just cross check.

Reply:
Removed this as this is not required now.